kafka-connect/s2i/resources/openshift-template.yaml

```diff
@@ -3,6 +3,31 @@ kind: Template
 metadata:
   name: barnabas-connect-s2i
 parameters:
+- description: Specifies the number of Kafka Connect instances to be started by default.
+  displayName: Number of Kafka connect instances
+  name: INSTANCES
+  required: true
+  value: "3"
+- description: A list of host:port pairs to use for establishing the initial connection to the Kafka cluster.
+  displayName: Kafka bootstrap servers
+  name: KAFKA_CONNECT_BOOTSTRAP_SERVERS
+  required: true
+  value: kafka:9092
+- description: A unique string that identifies the Connect cluster group this worker belongs to. Note this must not conflict with any consumer group IDs.
```

Comment:
long line

Reply:
Fixed. At least assuming you meant it as "too long in the YAML file".